global/scripts/mppmodecheck.shlib

```diff
@@ -0,0 +1,231 @@
+        T2wRestoreImage="NONE"
+    else
+      log_Msg "-> T2w image present (HCPStyleData)"
+    fi
+}
+
+
+check_mpp_fMRIVolume()
+{
+    # -- Slice timing correction
+
+    if [ ! "${DoSliceTimeCorrection}" == 'NONE' ]; then
+      log_Msg "-> Set to run slice timing correction (LegacyStyleData)"
+      Compliance="LegacyStyleData"
+    else
+      log_Msg "-> Not running slice timing correction (HCPStyleData)"
+    fi
+
+    # -- Use of BOLD scout image
+
+    if [ "${fMRIScout}" == 'NONE' ]; then
+      log_Msg "-> Set to use average motion corrected image as scout image (LegacyStyleData)"
+      Compliance="LegacyStyleData"
+    else
+      log_Msg "-> fMRIScout image is provided (HCPStyleData)"
+    fi
+
+    # -- Use of BOLD reference
+
+    if [ ! "${fMRIReference}" == 'NONE' ]; then
+      log_Msg "-> Set to use ${fMRIReference} as BOLD reference (LegacyStyleData)"
+      Compliance="LegacyStyleData"
+    else
+      log_Msg "-> Independent processing of BOLD image (HCPStyleData)"
+    fi
+
+    # -- Use of distortion correction methods
+
+    if [ "${DistortionCorrection}" == 'NONE' ]; then
+      log_Msg "-> Set not to use any distortion correction method (LegacyStyleData)"
+      Compliance="LegacyStyleData"
+    else
+      log_Msg "-> Distortion correction method specified (${DistortionCorrection}; HCPStyleData)"
+    fi
+
+    # -- Use of BOLD mask
+
+    if [ ! "${Mask}" == 'T1' ]; then
+      log_Msg "-> Set to use mask: ${Mask} (LegacyStyleData)"
+      Compliance="LegacyStyleData"
+    else
+      log_Msg "-> Set to use T1w image based mask (HCPStyleData)"
+    fi
+
+    # -- Preregister code
+
+    if [ ! "${Preregister}" == 'epi_reg' ]; then
+      Compliance="LegacyStyleData"
+      log_Msg "-> Set to use ${Preregister} for bold image pre-registration (LegacyStyleData)"
+    else
+      log_Msg "-> Set to use ${Preregister} for bold image pre-registration (HCPStyleData)"
+    fi
+
+    # -- BOLD registration to reference image
+
+    if [ ! "${ReferenceReg}" == 'linear' ]; then
+      Compliance="LegacyStyleData"
+      log_Msg "-> Set to use ${ReferenceReg} registration to reference image (LegacyStyleData)"
+    else
+      log_Msg "-> Set to use ${ReferenceReg} registration to reference image (HCPStyleData)"
+    fi 
+}
+
+check_mpp_fMRISurface()
+{
+    log_Msg "-> No LegacyStyleData functionality"
+}
+
+check_mpp_Diffusion()
+{
+    log_Msg "-> No LegacyStyleData functionality"   
+}
+
+
+
+#  --- Main function for checking compliance
+# 
+#  Use: check_mpp_compliance <pipeline name>
+
+
+check_mpp_compliance()
+{
+
+    # --- Define variables
+
+    local pipeline=$1
+    local hcpversion=`cat "$HCPPIPEDIR"/version.txt`
+    local Compliance="HCPStyleData"
+    local ispipeline="false"
+
+    # --- check and abort if pipeline name is not known
+
+    check_pipeline_name
+
+    if [ "${ispipeline}" == "false" ]; then
+        log_Err_Abort "${pipeline} is not a valid HCP Pipeline name. Aborting execution!"
```

Comment:
FWIW, `log_Err_Abort` already concludes with "ABORTING".  So, putting "Aborting execution!" into the message is redundant.

Reply:
Removed.